doc/examples/applications/plot_fluorescence_nuclear_envelope.py

```diff
@@ -150,15 +150,15 @@
 image_t_0_channel_1 = image_sequence[0, 1, :, :]
 selection = np.where(mask, image_t_0_channel_1, 0)
 
-fig, (ax0, ax1) = plt.subplots(1, 2, figsize=(12, 6), sharey=True)
+fig, ax = plt.subplots(1, 2, figsize=(12, 6), sharey=True)
```

Comment:
I would revert this, as explained previously, because the current code is correct and already OO style.

Reply:
Reverted